src/Compilers/CSharp/Test/Symbol/Symbols/TypeResolutionTests.cs

```diff
@@ -99,6 +99,7 @@ public class TestClass
             Assert.Same(c1TestClassT, c4.GetTypeByMetadataName("System.TestClass`1"));
         }
 
+#pragma warning disable CA1052 // Static holder types should be Static or NotInheritable
```

Comment:
💭 I almost feel like the attribute-based suppression would be better here

Reply:
I'm curious why?
I almost never used `SuppressMessage` attribute and instead always used `#pragma warning disable`. I'd love to know when one is preferred over the other.

Reply:
Reverted everything under src/Compilers.